manager/src/main/java/com/redhat/service/smartevents/manager/connectors/ConnectorsServiceImpl.java

```diff
@@ -51,7 +51,6 @@ public void createConnectorEntity(Processor processor, Action action) {
         persistConnectorEntity(processor, topicName, actionConnector.getConnectorType(), actionConnector.connectorPayload(action, topicName));
     }
 
-    @Transactional(Transactional.TxType.MANDATORY)
```

Comment:
@kostola Was there a reason why this is required?

Reply:
Well I got this warning message when starting the manager locally:

```
@Transactional(value = MANDATORY) will have no effect on method com.redhat.service.smartevents.manager.connectors.ConnectorsServiceImpl.persistConnectorEntity() because the method is private
```

so I removed it

Reply:
@robpblake The `private` method is invoked from a `public` method [with](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/a67296e1e6ee9711a85d82b1e0747c3fb95d7717/manager/src/main/java/com/redhat/service/smartevents/manager/connectors/ConnectorsServiceImpl.java#L42) the annotation; so I think we're OK.

@kostola a `private` method cannot be proxied by the codegen for the annotation.. so it is disregarded. That said I think we're OK with your change, as I mention above.